.vscode/settings.json

```diff
@@ -45,5 +45,17 @@
     "source.organizeImports": "explicit"
   },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "explorer.fileNesting.enabled": true,
+  "explorer.fileNesting.patterns": {
+    "*.ts": "${capture}.js",
+    "*.js": "${capture}.js.map, ${capture}.min.js, ${capture}.d.ts",
+    "*.jsx": "${capture}.js",
+    "*.tsx": "${capture}.ts, ${capture}.stories.tsx",
+    "tsconfig.json": "tsconfig.*.json",
+    "package.json": "package-lock.json, yarn.lock, pnpm-lock.yaml, bun.lockb",
+    ".env": ".env.local, .env.development, .env.production, .env.test, .env.staging, .env.local.example, .env.development.example, .env.production.example, .env.test.example, .env.staging.example",
+    "firebase.json": ".firebaserc, firestore.indexes.json, firestore.rules, storage.rules",
+    ".prettierrc": ".prettierignore"
+  }
```

Comment:
Did you mean to put these into this project's settings? They seem like personal preferences to me.

Reply:
I noticed today you had these in your project and was envious. Now that I look closer at electrofishing, you must have them as a user preference. I'll pull it out.

Reply:
If nothing else, it would dry up the config files for our individual projects.